biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -6,7 +6,7 @@ export class BioBattery implements Battery {
 
   constructor(currentBatteryPower: Energy = 0, maxCapacity: Energy = 10) {
     if (!this.validateInputs(currentBatteryPower, maxCapacity)) {
-      throw new Error('Cannot create a battery with these values');
+      throw 'Cannot create a battery with these values';
```

Comment:
Why did you make this a string instead of an Error?  Our internal style guide (not sure about the one you're following) requires us to throw an Error instead of a string

Reply:
> Why did you make this a string instead of an Error? Our internal style guide (not sure about the one you're following) requires us to throw an Error instead of a string

These are the changes that I made locally. At first Jest was not recognizing the ```new Error('my_error_string')``` syntax. But I tested it today morning, and I made the changes and have updated the PR